portal-web/test/functional/com/liferay/portalweb/macros/GDPR.macro

```diff
@@ -275,8 +275,8 @@ definition {
 			rowEntry = "${rowEntry}");
 
 		AssertElementPresent(
-			key_portlet = "${applications}",
-			locator1 = "ProductMenu#PORTLET_ACTIVE");
+			locator1 = "TextInput#TITLE",
+			value1 = "${dmDocumentTitle}");
```

Comment:
Please double check if the variable dmDocumentTitle can be renamed as applicationsTitle, so it can be reused for other entries.If edit other kind of applications also show its tile, we can rename it to applicationsTitle, otherwise, we need update the macro name.

Reply:
Done